webextension/js/popup.js

```diff
@@ -782,6 +786,41 @@ Logic.registerPanel(P_CONTAINER_EDIT, {
     });
   },
 
+  showAssignedContainers(assignments) {
+    const assignmentPanel = document.getElementById("edit-sites-assigned");
+    const assignmentKeys = Object.keys(assignments);
+    assignmentPanel.hidden = !(assignmentKeys.length > 0);
+    if (assignments) {
+      const tableElement = assignmentPanel.querySelector("table > tbody");
+      while (tableElement.firstChild) {
+        tableElement.firstChild.remove();
+      }
+      assignmentKeys.forEach((siteKey) => {
+        const site = assignments[siteKey];
+        const trElement = document.createElement("tr");
+        const assumedUrl = `https://${site.hostname}`;
```

Comment:
Because we don't have the full URL in storage (at least for legacy storages) So I assume https and if it doesn't load we get the default url.